connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/ConnectAssertions.java

```diff
@@ -554,7 +554,7 @@ protected Optional<Boolean> checkConnectorState(
      * @param connectorState
      * @param numTasks the expected number of tasks
      * @param tasksState
-     * @return true if the connector and tasks are in RUNNING state; false otherwise
+     * @return true if the connector and tasks are in expected states; false otherwise
```

Comment:
```suggestion
     * @return true if the connector and tasks are in the expected state; false otherwise
```
This one needs to be updated too :)

Reply:
Oh I just committed your suggestion and didn't realise this. Thanks for pointing this out 👍